cleansweep/committees/models.py

```diff
@@ -275,6 +275,18 @@ def get_committee(type):
             return type.committees.filter_by(place_id=self.id).first() or Committee(self, type)
         return [get_committee(type) for type in committee_types]
 
+    def get_incharges(self):
+        """Returns incharge at a given place.
+        """
+        committee_slug = u'incharges'
+        incharges = []
+        c = self.get_committee(committee_slug)
+
+        if c and c.committee_members:
+          incharges = filter(lambda m: m.role.role == u'Incharges', c.committee_members)
+        return incharges
+
+
```

Comment:
This is the only troubling part. 

The committee slug will be different for different levels. The Role for Incharge could be different as well. 

I think it is better to take it from a config setting or the first available committee. Lets have a function to get the committee name and role. That'll return first available committee and first role from that, unless a config setting is specified. Lets use `WARD_MASTER_COMMITTEE` and `WARD_MASTER_COMMITTEE_ROLE` with WARD substituted with appropriate level name.

Also, In python it is idiomatic to use list comprehensions instead of map/filter. The usual practice is:

    incharges = [m for m in c.committee_members if m.role.role == 'Incharges']